src/main/scala/com/elarib/PartialSbtPlugin.scala

```diff
@@ -90,13 +100,11 @@ object PartialSbtPlugin extends AutoPlugin {
         val modulesWithPath: Seq[(ProjectRef, ResolvedProject)] =
           allProjectRefs.filter(_._2.base != baseDir)
 
-        val diffsFiles: Seq[sbt.File] = changeGetter.changes
+        val diffsFiles: Seq[sbt.File] = changeGetter.changes.filterNot(f => excludeFile(baseDir)(f, excludeFiles))
```

Comment:
`excludeFile(baseDir)(f, excludeFiles)` was really hard to parse, I hope the suggested changes will make it more readable,